dali/kernels/slice/slice_cpu.h

```diff
@@ -275,9 +278,94 @@ void SliceKernel(OutputType *output,
   }
 }
 
+
+/**
+ * @brief Implementation of slice kernel with an execution engine.
+ */
+template <typename ExecutionEngine, typename OutputType, typename InputType, int Dims>
+void SliceKernel(ExecutionEngine &exec_engine,
+                 OutputType *output,
+                 const InputType *input,
+                 const TensorShape<Dims> &in_strides,
+                 const TensorShape<Dims> &out_strides,
+                 const TensorShape<Dims> &anchor,
+                 const TensorShape<Dims> &in_shape,
+                 const TensorShape<Dims> &out_shape,
+                 const OutputType *fill_values,
+                 int channel_dim = -1,  // negative if no channel dim or already processed
+                 int min_blk_sz = 16000,
+                 int req_nblocks = -1) {
+  // Parallelize
+  if (req_nblocks < 0)
+    req_nblocks = exec_engine.NumThreads() * 8;
```

Comment:
Why 8?

Reply:
Just a default value, in case no value is provided.